lib/index.js

```diff
@@ -1,23 +1,10 @@
 "use strict";
```

Comment:
Note that PouchDB itself is now a custom build. It's important to eat your own dog food. :smiley: